services/proxy/pkg/config/config.go

```diff
@@ -84,6 +85,12 @@ type AuthMiddleware struct {
 	CredentialsByUserAgent map[string]string `yaml:"credentials_by_user_agent"`
 }
 
+// PoliciesMiddleware configures the proxy policies middleware.
+type PoliciesMiddleware struct {
+	Enabled bool   `yaml:"enabled" env:"PROXY_POLICIES_ENABLED" desc:"Activates policy evaluation."`
```

Comment:
Compared when we do postprocessing, we need to briefly describe the difference of the usecases.
@kobergj had a good description but I cant remember - can you help?

Reply:
I wonder if we need this at all. Can't we just implictly activate it when `PROXY_POLICIES_QUERY` is not empty?

Reply:
> I wonder if we need this at all. Can't we just implictly activate it when PROXY_POLICIES_QUERY is not empty?

I would keep it as it gives the ability to enable/disable the functionality without the need to reconfigure it like for testing.

Reply:
i prefere the implicit activation, if no query is given, the middleware is disabled and vice versa